src/main/docs/guide/httpServer/errorHandling/exceptionHandler/builtInExceptionHandlers.adoc

```diff
@@ -19,7 +19,5 @@ Micronaut ships with several built-in handlers:
 | api:core.bind.exceptions.UnsatisfiedArgumentException[]
 | api:http.server.exceptions.UnsatisfiedArgumentHandler[]
 | api:web.router.exceptions.UnsatisfiedRouteException[]
-| api:http.server.netty.converters.UnsatisfiedRouteHandler[]
-| `org.grails.datastore.mapping.validation.ValidationException`
```

Comment:
We have this handler https://micronaut-projects.github.io/micronaut-groovy/latest/api/index.html  for gorm